guide/inheritance-wallet/landing-page.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: guide
+title: Inheritance wallet
+description: An in-depth UX reference design for a multi-key bitcoin wallet with inheritance features designed for families.
+nav_order: 6
+has_children: true
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/preview.jpg
+image_base: /assets/images/guide/inheritance-wallet/
```

Comment:
You can remove the `image_base`. It is only used in galleries to define which folder the gallery images are in. You don't use galleries, so you don't need to set this.